### PR TITLE
 update cookie acceptance action

### PR DESCRIPTION
**Pull request by doramatadora**
- updates the cookie banner "Accept" action to call the consent service (`consent.ft.com` is a separate Fastly service to `www.ft.com/__consent`, and we are deprecating the latter)
- bumps dependencies

### CODE REVIEW

File: src/components/cookie-consent/main.js [hunk 1]
```diff
@@ -18,26 +16,33 @@ module.exports = function customSetup (banner, done) {
...
 		const cookieBanner = bannerElem.closest('.n-ui-hide-enhanced');
-		if (cookieBanner)
+		if (cookieBanner) {
 			cookieBanner.classList.remove('n-ui-hide-enhanced');
+		}
 		const acceptForm = [].slice.call(wrapper.querySelectorAll('[data-action="accept-form"]'));
-		acceptForm.forEach(elem => elem.addEventListener('submit', (event) => updateConsent(elem, event), false));
+		acceptForm.forEach(elem =>
+			elem.addEventListener(
+				'submit',
```
Comment (jonnywyatt) [inline src/components/cookie-consent/main.js]: If it's a link now, why are you listening for submit?

File: src/components/cookie-consent/main.js [hunk 2]
```diff
@@ -18,26 +16,31 @@ module.exports = function customSetup (banner, done) {
...
 
 	const setup = () => {
 		const cookieBanner = bannerElem.closest('.n-ui-hide-enhanced');
-		if (cookieBanner)
+		if (cookieBanner) {
 			cookieBanner.classList.remove('n-ui-hide-enhanced');
-		const acceptForm = [].slice.call(wrapper.querySelectorAll('[data-action="accept-form"]'));
-		acceptForm.forEach(elem => elem.addEventListener('submit', (event) => updateConsent(elem, event), false));
+		}
+		const acceptButtons = wrapper.querySelectorAll('[data-action="accept-form"]');
```
Comment (jonnywyatt) [inline src/components/cookie-consent/main.js]: Minor, but there's only going to be one of these, could just use querySelector

Comment (debugwand) [reply src/components/cookie-consent/main.js]: yeah initially there were 2, now there is only 1

File: templates/partials/bottom/cookie-consent.html [hunk 3]
```diff
@@ -13,11 +12,7 @@ <h2 class="cookie-banner__head">Cookies on FT Sites</h2>
 				<a class="cookie-banner__link" href="https://www.ft.com/preferences/manage-cookies" data-trackable="onsite-message-link-cookie-consent" data-n-messaging-banner-action="">Manage cookies</a>
 			</div>
 			<div>
-				<form method="get" action="https://www.ft.com/__consent/consent-record-cookie"  data-action="accept-form">
-					<input type="hidden" name="redirect" value="{{#if ../realUrl}}{{../realUrl}}{{else}}https://www.ft.com/preferences/manage-cookies{{/if}}"
-					/>
-					<button type="submit" class="n-messaging-banner__button" data-n-messaging-banner-action="" data-trackable="onsite-message-button-cookie-consent">Accept &amp; continue</button>
-				</form>
+				<a href="https://consent.ft.com/__consent/consent-record-cookie?redirect={{#if @root.realUrl}}{{@root.realUrl}}{{else}}https://www.ft.com{{/if}}" class="n-messaging-banner__button" data-n-messaging-banner-action="" data-trackable="onsite-message-button-cookie-consent" data-action="accept-form">Accept &amp; continue</a>
```
Comment (pixelandpage) [inline templates/partials/bottom/cookie-consent.html]: Quick pedantic check: Should the `data-n-messaging-banner-action` def be an empty string?

Comment (doramatadora) [reply templates/partials/bottom/cookie-consent.html]: I'm not sure what this does, but as soon as our AB test reaches significance, this code will be superseded by `o-cookie-banner`.

**Pull request merged**